recipes/daggy/2.1/test_package/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.21.0)
+project(DaggyPackageTest)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(TARGETS)
```

So cmake generator won't mess the cmake targets generated by Conan